src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,58 @@
+.calendar {
+  $root: &;
+
+  width: $calendar-width;
+  padding-left: $calendar-block-padding;
+  padding-top: $calendar-block-padding * 2; 
+  // Doubled padding-top, 
+  // so the top row don't go over top of the page on hover.
```

Comment:
remove comments
PS. you can just use different paddings and not use any variables for them :)